storage/storage.go

```diff
@@ -108,31 +109,55 @@ func (m multiStorage) Stat(ctx context.Context, element ingest.Identifiable) (*O
 	return o0, err.Err()
 }
 
-func (m multiStorage) Store(ctx context.Context, element ingest.Identifiable, download func(context.Context, ingest.Identifiable) (ingest.Object, error)) (*url.URL, error) {
+func (m multiStorage) Configure(map[string]any) error {
+	return errors.New("not implemented")
+}
+
```

Comment:
This this really needed? `multiStorage` only needs to implement the `Storage` interface and not the `Destination` interface, right?
```suggestion
```